dataflux_pytorch/dataflux_mapstyle_dataset.py

```diff
@@ -115,10 +124,11 @@ def __init__(
             None.
         """
         super().__init__()
+        if storage_client is not None and OS != LINUX:
```

Comment:
For these checks, rather than checking the OS, would it make more sense to check the set multiprocessing start_method instead? I think that would be more reliable, since even Linux users might use "spawn" in some cases https://docs.python.org/3/library/multiprocessing.html#multiprocessing.get_start_method